### PR TITLE
Update w/ open peer review policies

### PR DESCRIPTION
**Pull request by FofResearch**

### CODE REVIEW

**Comment by jpolka** [comment]
Thanks @garymcdowell! @dhimmel , any idea why these checks failed?

**Comment by dhimmel** [comment]
The [build error](https://travis-ci.com/transpose-publishing/policies-database/builds/73091855#L809) is:

```
ruamel.yaml.scanner.ScannerError: mapping values are not allowed here
```

I believe this is what we saw last time when there was an error with the YAML formatting. My guess is there is an unquoted `: `.

File: policies/policies/romeo_2082.yml [hunk 1]
```diff
@@ -22,31 +22,31 @@ child-policies: []
...
-identities-published:
+identities-published: no
 
 # Co-reviewer policy url (valid url)
-co-review-url:
+co-review-url: http://dev.biologists.org/content/reviewer-guide#info
 
 # Journal’s policy about co-reviewers - ie people who collaborate with an invited reviewer (free text)
-co-review-policy:
+co-review-policy: We simply ask that: the name of the co-reviewer is reported to the Editor (a field is provided in the report form for this purpose); the same rules of confidentiality and conflict of interest be applied; there is a genuine mentoring process; the senior invited reviewer takes responsibility for the report delivered to the journal.
```
Comment (dhimmel) [inline policies/policies/romeo_2082.yml]: Try putting this in "quotes"

File: policies/policies/romeo_2082.yml [hunk 2]
```diff
@@ -22,31 +22,31 @@ child-policies: []
...
 
 # Does the journal make it clear in the reviewer invitation email that co-reviewers can contribute? (yes/no)
-co-review-invited:
+co-review-invited:
```
Comment (dhimmel) [inline policies/policies/romeo_2082.yml]: If editing this file, you might as well delete these spaces, so we don't touch lines unless we're changing them.

**Comment by jpolka** [comment]
Thanks @dhimmel!

**Pull request merged**